pandas/core/arrays/interval.py

```diff
@@ -1015,6 +1017,40 @@ def repeat(self, repeats, **kwargs):
         right_repeat = self.right.repeat(repeats, **kwargs)
         return self._shallow_copy(left=left_repeat, right=right_repeat)
 
+    _interval_shared_docs['overlaps'] = """\
+        Check elementwise whether the given input overlaps the intervals in
+        the %(klass)s.
+
+        .. versionadded:: 0.24.0
+
+        Parameters
+        ----------
+        other : Interval-like
+            Interval-like object to check against for an overlap.
+
+        Returns
+        -------
+        ndarray
+            Boolean array positionally indicating where an overlap occurs.
+    """
```

Comment:
Can you add some examples to the docstring

Reply:
done